messages/messages.json

```diff
@@ -29,5 +29,8 @@
   "noPromptAuth": "do not prompt for auth confirmation in demo mode",
   "disableMasking": "disable masking of user input (for use with problematic terminals)",
   "clientSecretStdin": "OAuth client secret of personal connected app?",
-  "invalidInstanceUrl": "Invalid instance URL. Specify a Salesforce instance URL using the format <domainname>.salesforce.com"
+  "invalidInstanceUrl": "Invalid instance URL. Specify a Salesforce instance URL using the format <domainname>.salesforce.com",
+  "accessTokenStdin": "Access token of user to use for authentication",
+  "noPrompt": "do not prompt for confirmation",
+  "overwriteAuthFileYesNo": "An authorization file exists for user \"%s\". Are you sure you want to save the file?\n\nSave the user (y/n)?"
```

Comment:
I'm a little unclear about these two questions.  Are you asking whether the user wants to overwrite a file?  If so, maybe ask this as the first question:

"Are you sure you want to overwrite the existing file?"

And I don't know what "Save the user" means.

Reply:
@jshackell-sfdc I like that better. Will change.

Reply:
@jshackell-sfdc I am reusing an existing message here. If I change it, it will affect the other commands as well.

Reply:
Got it.  Let's just leave it.   Thx

Reply:
Although, I still think it'll be confusing to the user.   But I also don't want to suggest changing a message that applies to other commands without knowing what those commands are.  Could you create a new message for just this situation?  I don't know how you all deal with these things.  I leave it up to you.  With the caveat that I'm not thrilled with the message if you leave it as-is :)

Reply:
I will add a new message.